Jenkinsfile

```diff
@@ -183,6 +176,31 @@ pipeline {
                         }
                     }
                 }
+                stage ("Upload to Artifactory") {
+                    when {
+                        anyOf {
+                            expression { return params.FORCE_PYUPLOAD }
+                            expression {
+                                bbcShouldUploadArtifacts(branches: ["dev"])
+                            }
+                        }
+                    }
+                    steps {
+                        script {
+                            env.artifactoryUpload_result = "FAILURE"
+                        }
+                        bbcGithubNotify(context: "artifactory/upload", status: "PENDING")
+                        bbcTwineUpload(toxenv: "py36")
```

Comment:
Does `bbcMakeGlobalWheel` need to be called when on the `dev` branch? Maybe this should be done in a separate stage?

It appears that `bbcMakeGlobalWheel` will only be called in the previous stage when on the `master` branch or when `FORCE_PYUPLOAD` is set.